generate.js

```diff
@@ -1,31 +1,29 @@
 'use strict';
-var fs = require('fs');
-var Download = require('download');
-var CSV = require('comma-separated-values');
-var arrayUniq = require('array-uniq');
+const fs = require('fs');
+const download = require('download');
+const CSV = require('comma-separated-values');
+const arrayUniq = require('array-uniq');
 
-new Download({extract: true})
-	.get('http://www.mapcode.com/kader/isotables.zip')
-	.run(function (err, files) {
-		var data = files[0].contents.toString();
-		var items = new CSV(data, {header: true}).parse();
-		var ret = [];
+download('http://www.mapcode.com/kader/isotables.zip', {extract: true}).then(files => {
+	const data = files[0].data.toString();
+	const items = new CSV(data, {header: true}).parse();
```

Comment:
I just saw that. Yeah, that's dumb. I guess it's easier to just stick with the CSV then.

Could you switch to use https://github.com/sindresorhus/neat-csv ?